pkg/types/ovirt/platform.go

```diff
@@ -33,4 +33,16 @@ type Platform struct {
 	// Default will set the image field to the latest RHCOS image.
 	// +optional
 	DefaultMachinePlatform *MachinePool `json:"defaultMachinePlatform,omitempty"`
+
+	// AffinityGroupsNames contains the names of the RHV affinity groups that the machines will join
+	// +optional
+	AffinityGroups []*AffinityGroup `json:"affinityGroups"`
```

Comment:
This does not need to be a slice of pointers. The extra indirection adds unnecessary complexity.
```
	AffinityGroups []AffinityGroup `json:"affinityGroups"`
```

Reply:
Done